packages/alchemy/src/react/hooks/useExportAccount.ts

```diff
@@ -0,0 +1,80 @@
+"use client";
+
+import { useMutation, type UseMutateFunction } from "@tanstack/react-query";
+import { createElement, useCallback } from "react";
+import type { ExportWalletParams as ExportAccountParams } from "../../index.js";
+import { useAlchemyAccountContext } from "../context.js";
+import type { BaseHookMutationArgs } from "../types.js";
+import { useSigner } from "./useSigner.js";
+
+export type UseExportAccountData = boolean;
+
+export type UseExportAccountParams = void;
+
+export type UseExportAccountMutationArgs = BaseHookMutationArgs<
+  UseExportAccountData,
+  UseExportAccountParams
+>;
+
+export type ExportAccountComponentProps = {
+  iframeCss?: string;
+  className?: string;
+  isExported: boolean;
+};
+
+export type UseExportAccountResult = {
+  exportAccount: UseMutateFunction<
+    UseExportAccountData,
+    Error,
+    UseExportAccountParams,
+    unknown
+  >;
+  isExported: boolean;
+  isExporting: boolean;
+  error: Error | null;
+  ExportAccountComponent: (props: ExportAccountComponentProps) => JSX.Element;
+};
+
+export function useExportAccount(
+  params: ExportAccountParams,
+  mutationArgs?: UseExportAccountMutationArgs
+): UseExportAccountResult {
```

Comment:
oh one more nit, let's be consistent here as we are https://github.com/alchemyplatform/aa-sdk/pull/568/files#diff-80d22c5bbe3df8f9522514eafb6c6a7b37485af1b0236ddc8ec14f123fad10dbR45-R48

Reply:
great catch